docs/usage/configuration-options.md

```diff
@@ -10,16 +10,20 @@ Any config you define applies to the whole repository (e.g. if you have a monore
 
 You can store your Renovate configuration file in one of the following locations:
 
-- `.github/renovate.json`
-- `.github/renovate.json5`
-- `.gitlab/renovate.json`
-- `.gitlab/renovate.json5`
-- `.renovaterc.json`
-- `renovate.json`
-- `renovate.json5`
-- `.renovaterc`
+- renovate.json
+- renovate.json5
+- .github/renovate.json
+- .github/renovate.json5
+- .gitlab/renovate.json
+- .gitlab/renovate.json5
+- .renovaterc
+- .renovaterc.json
 - `package.json` _(within a `"renovate"` section)_
 
+_Note_: Storing Renovate config in `package.json` is now deprecated and its support may be removed in the future.
+
+When renovating a repository, Renovate will try to autodetect/touch those files in exactly the same order they are listead above.
```

Comment:
```suggestion
When renovating a repository, Renovate will try to detect the configuration files in the order listed above.
```

How about this sentence?

Does Renovate `touch`/edit/change its own configuration files? I think that is something that only the end-user does?

Reply:
Renovate does not touch them, this sentence was incorrect